src/components/textfield/TextField.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Please import the props like this:

```
import { bool, string } from 'prop-types';
```

We are trying to have consistency in our codebase.